packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/risk-management.tsx

```diff
@@ -25,8 +27,22 @@ const RiskManagement = observer(({ is_minimized }: TTradeParametersProps) => {
         is_market_closed,
         take_profit,
         stop_loss,
+        proposal_info,
+        validation_errors,
+        trade_types,
+        contract_type,
+        trade_type_tab,
     } = useTraderStore();
 
+    const contract_types = getDisplayedContractTypes(trade_types, contract_type, trade_type_tab);
+
+    const { is_error_matching_trade_param: has_error } = useTradeParamError({
```

Comment:
For scenarios, when we haven't change the TP or SL, but already have an error for their fields (e.g. in case if Stake was changed).